node/pkg/watchers/aptos/watcher.go

```diff
@@ -124,9 +287,9 @@ func (e *Watcher) observeData(logger *zap.Logger, data gjson.Result, native_seq
 		return
 	}
 
-	consistency_level := data.Get("consistency_level")
-	if !consistency_level.Exists() {
-		logger.Error("consistency_level field missing")
+	consistencyLevel := data.Get("consistencyLevel")
```

Comment:
Sorry if this is a dumb question, but doesn't the parameter to Get() have to match the json tag? If so, will changing it to camel case break things?